plugins/xml-element-extractor/scripts/extract_xml_element.py

```
@@ -0,0 +1,248 @@
+
+
+def format_xml_with_python(file_path):
+    """Format XML file using Python's minidom as fallback."""
+    try:
+        print("Formatting XML with Python (fallback)...")
+
+        with open(file_path, 'r', encoding='utf-8') as f:
+            content = f.read()
+
+        # Parse and format with minidom
+        dom = minidom.parseString(content)
```

Ruff (0.14.3)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S318)